client/SDL/SDL3/sdl_clip.cpp

```diff
@@ -671,27 +702,45 @@ const void* sdlClip::ClipDataCb(void* userdata, const char* mime_type, size_t* s
 	}
 
 	{
-		HANDLE hdl[2] = { clip->_event, freerdp_abort_event(clip->_sdl->context()) };
-		auto status = WaitForMultipleObjects(ARRAYSIZE(hdl), hdl, FALSE, INFINITE);
-		if (status != WAIT_OBJECT_0)
+		HANDLE hdl[2] = { freerdp_abort_event(clip->_sdl->context()), clip->_event };
+		DWORD status = WaitForMultipleObjects(ARRAYSIZE(hdl), hdl, FALSE, 10 * 1000);
```

Comment:
not sure a hard coded timeout is a good idea, got reports of people using `RDP` over satellite :grin: 
anyway, small nitpick for later cleanup.

Reply:
Yeah but here we're in the SDL callback so on the other hand we can't stay here forever...

Reply:
true.
lets leave it at that and wait for the complaints :grin: